src/ui/server/controllers/insurance/your-buyer/connection-with-buyer/validation/rules/connection-with-buyer-description.ts

```diff
@@ -29,7 +29,6 @@ const MAXIMUM = 1000;
  * @returns {Object} Validation errors
  */
 const connectionWithBuyerDescriptionRule = (formBody: RequestBody, errors: object) => {
-  // if HAS_DIFFERENT_TRADING_NAME radio is yes then check validation
```

Comment:
are these comments not needed?

Reply:
I don't think so, the code is self explanatory:
```js
if (formBody[CONNECTION_WITH_BUYER] === 'true') {
```